### PR TITLE
Design

### PR DESCRIPTION
**Pull request by fdo2**
Done with login page design

### CODE REVIEW

File: src/views/layouts/main.hbs [hunk 1]
```diff
@@ -3,6 +3,9 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://unpkg.com/tachyons@4.10.0/css/tachyons.min.css"/>
+    <link rel="stylesheet" href="style.css">
+    <img src="./logo.jpg" class="ml5 cover pa2">
```
Comment (jema28) [inline src/views/layouts/main.hbs]: Remove the margin here and put flex here instead!

Comment (fdo2) [reply src/views/layouts/main.hbs]: Ok :smiley_cat:

File: public/style.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  flex-direction: column;
+  align-items: center;
+}
+
+.registerForm{
+  display: flex;
+  flex-direction: column;
+}
+
+::placeholder{
```
Comment (jema28) [inline public/style.css]: You need to add input here so it's clear which placeholder you want to change.

Comment (fdo2) [reply public/style.css]: :+1:

**Pull request merged**